doc/redirects.json

```diff
@@ -3425,15 +3425,6 @@
   "javascript-using-node_modules": [
     "index.html#javascript-using-node_modules"
   ],
-  "javascript-packages-nixpkgs": [
-    "index.html#javascript-packages-nixpkgs"
-  ],
-  "javascript-adding-or-updating-packages": [
-    "index.html#javascript-adding-or-updating-packages"
-  ],
-  "javascript-git-error": [
-    "index.html#javascript-git-error"
-  ],
```

Comment:
These should've been appended to the list of `javascript-tool-specific`. Removing identifiers entirely beats the point of having this setup.

Reply:
It's designed with these situations in mind:
- Heading was renamed without moving the content
- Content moved with little to no modification elsewhere
- Content removed and users are urged to use a shiny new thing
- Content removed without a replacement (now points to release notes entry)

READMEs within the repository with docs is something nixpkgs moved off of, right? Which is why the current system doesn't support redirecting to it without plans to do it in the future. It would also be a significant technical effort to accomplish that.